spec/jobs/application_job_spec.rb

```diff
@@ -1,9 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe ApplicationJob, type: :job do
-  context 'warning error classes' do
-    it 'returns empty class' do
-      expect(described_class.warning_error_classes.empty?).to be true
+  describe '.warning_error_classes' do
+    it 'is empty by default' do
+      expect(described_class.warning_error_classes).to be_empty
```

Comment:
My goals were:
1. clarify in the describe which method
2. specify we expect an empty container, not an empty class